CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## Supported Release 2.2.0
```

Comment:
If the only change is Rubocop, should this be 2.1.1?

Reply:
I'm of the opinion that the sheer mass of minor changes to the code qualify it for a y release.